### PR TITLE
docs(button): improve docs and shape remove undeclared doc type

### PR DESCRIPTION
**Pull request by hengkx**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [x] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  Button shape remove undeclared doc type.         |
| 🇨🇳 Chinese |  Button shape 移除文档中未声明的类型。         |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/button/index.en-US.md](https://github.com/ant-design/ant-design/blob/improve/doc/components/button/index.en-US.md)
[View rendered components/button/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/improve/doc/components/button/index.zh-CN.md)

### CODE REVIEW

File: components/button/button.tsx [hunk 1]
```diff
@@ -74,7 +74,7 @@ function spaceChildren(children: React.ReactNode, needInserted: boolean) {
 
 const ButtonTypes = tuple('default', 'primary', 'ghost', 'dashed', 'link', 'text');
 export type ButtonType = typeof ButtonTypes[number];
-const ButtonShapes = tuple('circle', 'circle-outline', 'round');
```
Comment (xrkffgg) [inline components/button/button.tsx:77]: 这个没了？

Comment (hengkx) [reply components/button/button.tsx:77]: 文档本来 没有，我看了下代码 'circle', 'circle-outline'这两个代码是一样的效果，而且是5年前写的，只是 剔除了ts声明 样式没动，对用户没有影响的

Comment (xrkffgg) [reply components/button/button.tsx:77]: 我试了一下，这个还真有

Comment (hengkx) [reply components/button/button.tsx:77]: image
还有什么地方？

Comment (xrkffgg) [reply components/button/button.tsx:77]: 我用 Demo 试了下，效果不一样

Comment (hengkx) [reply components/button/button.tsx:77]: 截个图 或 链接？

Comment (hengkx) [reply components/button/button.tsx:77]: https://codesandbox.io/s/nervous-ishizaka-38277?file=/index.js:845-852     我这边看效果是一样的

Comment (xrkffgg) [reply components/button/button.tsx:77]: image

椭圆的

Comment (hengkx) [reply components/button/button.tsx:77]: image
这个也加上文字 看下

Comment (xrkffgg) [reply components/button/button.tsx:77]: 噢，原来这样 😂😂

**Comment by afc163** [comment]
css 也干掉吧。

**Comment by hengkx** [comment]
> css 也干掉吧。

removed.

**Pull request merged**